tests.py

```diff
@@ -444,6 +455,36 @@ def test_random_ind_2():
 
     assert expected_code == export_pipeline(pipeline, tpot_obj.operators, tpot_obj._pset)
 
+def test_pipeline_score_save():
+    """Assert that the TPOTClassifier can generate the same pipeline export with random seed of 39."""
+    tpot_obj = TPOTClassifier(random_state=39)
+    tpot_obj._pbar = tqdm(total=1, disable=True)
+    pipeline = tpot_obj._toolbox.individual()
+    expected_code = """import numpy as np
+
+from sklearn.feature_selection import SelectPercentile, f_classif
+from sklearn.model_selection import train_test_split
+from sklearn.pipeline import make_pipeline
+from sklearn.tree import DecisionTreeClassifier
+
+# NOTE: Make sure that the class is labeled 'class' in the data file
+tpot_data = np.recfromcsv('PATH/TO/DATA/FILE', delimiter='COLUMN_SEPARATOR', dtype=np.float64)
+features = np.delete(tpot_data.view(np.float64).reshape(tpot_data.size, -1), tpot_data.dtype.names.index('class'), axis=1)
+training_features, testing_features, training_target, testing_target = \\
+    train_test_split(features, tpot_data['class'], random_state=42)
+
+# Score on the training set was:0.929813743
+exported_pipeline = make_pipeline(
+    SelectPercentile(score_func=f_classif, percentile=65),
+    DecisionTreeClassifier(criterion="gini", max_depth=7, min_samples_leaf=4, min_samples_split=18)
+)
+
+exported_pipeline.fit(training_features, training_target)
+results = exported_pipeline.predict(testing_features)
+"""
+
+    assert_equal(expected_code, export_pipeline(pipeline, tpot_obj.operators, tpot_obj._pset, pscore=0.929813743))
+
```

Comment:
I think these tests look good, but they need to be rebased on the most recent merge. @teaearlgraycold reorganized `tests.py` into a `tests` directory. Less clutter.